packages/devextreme/js/core/component.js

```diff
@@ -12,6 +12,7 @@ import { PostponedOperations } from './postponed_operations';
 import { isFunction, isPlainObject, isDefined } from './utils/type';
 import { noop } from './utils/common';
 import { getPathParts } from './utils/data';
+import license from '../__internal/core/license/license_validation';
```

Comment:
Can we use named import?
```suggestion
import { verifyLicense } from '../__internal/core/license/license_validation';
```

Reply:
We can. But in that case we cant use sinon.spy/stub in tests.